main.go

```diff
@@ -25,31 +30,44 @@ var (
 )
 
 func main() {
+	ctx := context.Background()
+	if err := run(ctx); err != nil {
+		log.Event(ctx, "unable to run application", log.Error(err))
+		os.Exit(exitFail)
+	}
+}
+
+func run(ctx context.Context) error {
 	log.Namespace = "dp-frontend-cookie-controller"
+
+	signals := make(chan os.Signal, exitFail)
```

Comment:
You shouldn't be using the exitFail value to set the buffer on the channel and this should be set to `1`